maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -52,8 +50,6 @@ eBPF mode.  Below, we give a couple of options for how to get the cluster runnin
 <label:Bottlerocket,active:true>
 <%
 
-#### Option 1: Bottlerocket
-
 The easiest way to start an EKS cluster that meets eBPF mode's requirements is to use Amazon's 
 [Bottlerocket](https://aws.amazon.com/bottlerocket/) OS, instead of the default.  Bottlerocket is a 
 container-optimised OS with an emphasis on security; it has a recent enough kernel to use eBPF mode.
```

Comment:
it has a version of the kernel works for eBPF mode.

Reply:
How about: 
```suggestion
container-optimised OS with an emphasis on security; it has a version of the kernel which is compatible with eBPF mode.
```